src/components/common/datePicker/CorrectionCustomHeader.tsx

```diff
@@ -1,75 +1,87 @@
 import styled from '@emotion/styled';
 
-import ArrangeBtn from '../arrangeBtn/ArrangeBtn';
-import TextboxInput from '../textbox/TextboxInput';
-
-import formatDatetoString from '@/utils/formatDatetoString';
+import Button from '../v2/button/Button';
+import IconButton from '../v2/IconButton';
+import MainDate from '../v2/TextBox/MainDate';
 
 interface CustomHeaderProps {
-	prevDate: Date | null;
+	selected: Date | null;
 	decreaseMonth: () => void;
 	increaseMonth: () => void;
 	prevMonthButtonDisabled: boolean;
 	nextMonthButtonDisabled: boolean;
 	onChange: (date: Date | null) => void;
-	dateTextRef: React.RefObject<HTMLInputElement>;
+	onClose: () => void;
 }
 
 function CorrectionCustomHeader({
-	prevDate,
+	selected,
 	decreaseMonth,
 	increaseMonth,
 	prevMonthButtonDisabled,
 	nextMonthButtonDisabled,
 	onChange,
-	dateTextRef,
+	onClose,
 }: CustomHeaderProps) {
+	const selectedDate = selected || new Date();
+	const today = new Date();
 	return (
-		<div className="react-datepicker__header-custom">
-			<InfoBox>
-				<TextboxInput
-					variant="date"
-					placeholder={formatDatetoString(prevDate)}
-					onDateChange={onChange}
-					dateTextRef={dateTextRef}
+		<HeaderLayout className="react-datepicker__header-custom">
+			<IconButton iconName="IcnX" size="small" type="normal" disabled={false} onClick={onClose} />
+			<DateWrapper>
+				<MainDate
+					month={selectedDate.getUTCMonth() + 1}
+					day={selectedDate.getUTCDate()}
+					year={selectedDate.getUTCFullYear()}
 				/>
```

Comment:
피그마 상에는 day가 안떠서 수정 요청드립니다!!

```suggestion
				<MainDate
					month={selectedDate.getUTCMonth() + 1}
					year={selectedDate.getUTCFullYear()}
				/>
```

Reply:
아하 먼가 낀다했더니 그거였군요ㅜㅜㅋㅋ 꼼꼼왕 감사합니당